scripts/lib/bump_version.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+from typing import Union
+
+import semver
```

Comment:
You are using semver which is currently development lib with Warning, would be worth mentioning somewhere (at least in the exporters/requirements-dev.txt):
https://github.com/python-semver/python-semver/blob/master/README.rst

Reply:
Agree, but let's do that in smaller separate PR. #359